spec.bs

```diff
@@ -2907,7 +2907,10 @@ dictionary ReportWinBrowserSignals : ReportingBrowserSignals {
   DOMString buyerReportingId;
   unsigned short modelingSignals;
   unsigned long dataVersion;
+  KAnonStatus kAnonStatus;
 };
+
+enum KAnonStatus { "passedAndEnforced", "same-passedNotEnforced", "belowThreshold", "notCalculated" };

```

Comment:
"same-passedNotEnforced" does not match any of the options below

Reply:
Typo fixed.